runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/client/SdkClientOption.kt

```diff
@@ -32,6 +33,11 @@ public object SdkClientOption {
      * The client logging mode (see [SdkLogMode]
      */
     public val LogMode: ClientOption<SdkLogMode> = ClientOption("LogMode")
+
+    /**
+     * The source of time for an operation
+     */
+    public val Clock: ClientOption<Clock> = ClientOption("Clock")
```

Comment:
**Question**: While I see the benefit of a specifiable `Clock` to facilitate testing, I don't understand the use case for a `ClientOption` for it. What do you see this providing?

Reply:
We can remove it for now probably, was just thinking it would be convenient to be able to specify the source of time for an operation. Until we have further need for this though I think we can just isolate it to where it is used for now.